.evergreen/deploy.yml

```diff
@@ -4,7 +4,6 @@ functions:
       params:
         working_dir: ui/${app_dir}
```

Comment:
I believe we could use `add_expansions_to_env` instead of copying all of these 👀

Reply:
done and checked that sentry reporting was working on staging